src/components/setting/setting.js

```diff
@@ -1,151 +1,201 @@
+                  <span className={styles.labelName}>{t('Second Passphrase')}</span>
+                  <p>
+                    {t('Every time you make a transaction you’ll need to enter your second passphrase in order to confirm it.')}
+                  </p>
+                  <p className={styles.highlight}>{t('Once activated can’t be turned off.')}</p>
+                  {!hasSecondPassphrase ?
+                    <SignInTooltipWrapper>
+                      <Link
+                        className={`register-second-passphrase ${styles.link}`}
+                        to={`${routes.secondPassphrase.path}`}>
+                        {t('Activate (5 LSK Fee)')}
+                      </Link>
+                    </SignInTooltipWrapper>
+                  : (
+                    <span className={`${styles.activeLabel} second-passphrase-registered`}>
+                      {t('Second Passphrase is active for this account.')}
+                    </span>
+                  )}
+                </div>
+              </div>
+            </section>
+            <section>
+              <h1>{t('Advanced')}</h1>
+              <label className={`${styles.fieldGroup} ${styles.checkboxField}`}>
+                <CheckBox
+                  name={'showNetwork'}
+                  className={`${styles.checkbox} showNetwork`}
+                  checked={settings.showNetwork}
+                  onChange={this.handleCheckboxChange}
+                />
+                <div>
+                  <span className={styles.labelName}>{t('Network switcher')}</span>
+                  <p>{t('Enable a network switcher that lets you select testnet or custom node when logging in.')}</p>
+                </div>
+              </label>
+              {localStorage.getItem('btc') ? ( // TODO: Remove when enabling BTC for final user
+              <label className={`${styles.fieldGroup} ${styles.checkboxField}`}>
+                <CheckBox
+                  name={'BTC'}
+                  className={`${styles.checkbox} enableBTC`}
+                  checked={!!(settings.token && settings.token.list.BTC)}
+                  onChange={this.handleTokenToggle}
+                />
+                <div>
+                  <span className={styles.labelName}>{t('BTC token')}</span>
+                  <p>{t('By enabling it, you will be able to manage your BTC tokens inside the application.')}</p>
+                </div>
+              </label>
+              ) : null}
+              <label className={`${styles.fieldGroup} ${styles.checkboxField}`}>
+                <CheckBox
+                  name={'advancedMode'}
+                  className={`${styles.checkbox} advancedMode`}
+                  checked={settings.advancedMode}
+                  onChange={this.handleCheckboxChange}
+                />
+                <div>
+                  <span className={styles.labelName}>{t('Delegate Features')}</span>
+                </div>
+              </label>
```

Comment:
Now that https://github.com/LiskHQ/lisk-hub/pull/2080 is merged, 'Delegate Features' should be removed from settings, as it's always displayed.

Reply:
Done